portablemc/cli/lang.py

```diff
@@ -189,6 +192,7 @@ def get(key: str, **kwargs) -> str:
     "search.flags.local": "local",
     "search.flags.stable": "stable",
     "search.loader_version": "Loader version",
+    "search.OptiFine.ForgeCompatibility": "Forge compatibility",
```

Comment:
Please use lowercase `snake_case` for this key, like `seach.optifine.forge_compatibility`, remember to change it where it's used.